Userland/Libraries/LibGfx/Filters/StackBlurFilter.cpp

```diff
@@ -284,7 +284,7 @@ FLATTEN void StackBlurFilter::process_rgba(u8 radius)
             if (alpha != 0)
                 set_pixel(x, y, Color((red_sum * sum_mult) >> sum_shift, (green_sum * sum_mult) >> sum_shift, (blue_sum * sum_mult) >> sum_shift, alpha));
             else
-                set_pixel(x, y, transparent_white);
+                set_pixel(x, y, fill_color);
```

Comment:
wouldn't it be reasonable to just not set a color in this case?

Reply:
Not necessarily I think it is possible to blur to nothing with a high enough radius (and removing these makes no difference in the profile, so I see no reason to).